The Shipyard.txt

```diff
@@ -0,0 +1,58 @@
+
+
+
+
+
+package gameComponents;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+public class Ships {
+public class Faction {
+
```

Comment:
A few things happening here.

1) Did you intend to include this new file here in the same PR as the ship text?  I had expected they would be separate branches and separate pull requests so that when the text file was done it could be merged.  If these 2 stay in the same pull request/branch, *which is fine* then they won't be merged for a while.

2) I'm guessing you didn't make this file in your IDE, its missing a few closing braces, and nesting classes like this is weird.  Was that intentional?  If so, what was the reasoning?

3) Wrong file name/extension.  I'm about 95% sure that the class and file name should be the same, ie if this file defines the Ships class, the file should be Ships.java, or if it is the ShipFactory class it would be ShipFactory.java.  IntelliJ should do that automatically for you.   Just right click on the folder you want your class -> New -> Java Class, and type in your name.

Reply:
Let me make this a separate thing